cmd/snap-confine/snap-confine.c

```diff
@@ -523,10 +531,25 @@ int main(int argc, char **argv)
 	// Now that we've dropped and regained SYS_ADMIN, we can load the
 	// seccomp profiles.
 	if (sc_apply_seccomp_profile_for_security_tag(invocation.security_tag)) {
+		sc_explain_li("Applied seccomp profile: %s",
+			   invocation.security_tag);
+                sc_explain_start_section("");
```

Comment:
still sections started with "" are a bit odd, we probably need something else to take care of this

Reply:
Yeah, I think that's right, I removed the format from "sc_start_section" now to just make it indent.